tools/ci/scripts/cve/third_party_libraries_cves_scanner.py

```diff
@@ -0,0 +1,383 @@
+
+    cves_per_library.append({"name": library_name, "cves": cves})
+
+if len(cves_per_library) == 0:
+    exit(0)
+
+# Always print the cves that have been found
+print("\nFound the following CVEs:")
+
+libraries_with_cves = []
+libraries_with_ignored_cves = []
+
+# Split ignored CVEs from still valid CVEs and organize both for display
+for library in cves_per_library:
+    cves = library["cves"]
+
+    cves_messages = []
+    ignored_cves_messages = []
+
+    for cve in cves:
+        print(f"Found CVE: {cve.name}")
+        if cve.name not in all_ignored_cves:
+            cves_messages.append(f"\t Name: {cve.name}\tSeverity: {cve.severity}")
+        else:
+            ignored_cves_messages.append(f"\t Name: {cve.name}\tSeverity: {cve.severity}")
+
+    if len(cves_messages) > 0:
+        libraries_with_cves.append((library["name"], cves_messages))
+
+    if len(ignored_cves_messages) > 0:
+        libraries_with_ignored_cves.append((library["name"], ignored_cves_messages))
+
+for library in libraries_with_cves:
+    print(f"Library: {library[0]}")
+    for cve in library[1]:
+        print(cve)
+    print()
+
+print("\nThe following CVEs have been ignored:")
+
+for library in libraries_with_ignored_cves:
+    print(f"Library: {library[0]}")
+    for cve in library[1]:
+        print(cve)
+    print()
+
+if args.create_issues:
+
+    github_token = None
+    if not args.github_token:
+        github_token = os.environ.get("GITHUB_TOKEN")
+
+        if github_token is None:
+            print("Missing github token")
+            exit(1)
+    else:
+        github_token = args.github_token
+
+    github_api = GithubAPI(args.source_repo, args.dest_repo, github_token, DEBUG)
+
+    print("Retrieving already opened issues")
+    issues_batches = github_api.getRecentOpenIssues()
+
+    # Process all opened issues and extract a list of CVEs that have been already reported
+    opened_cve_issues = set()
+    for batch in issues_batches:
+        for issue in batch:
+            if issue.get("pull_request") is not None:
+                continue
+
+            cve_id = parseCVEFromTitle(issue["title"])
+
+            if cve_id is None:
+                continue
+
+            opened_cve_issues.add(cve_id)
+
+    # Open issues for each new CVE
+    errors = 0
+    count = 0
+    print("Opening issues for new CVEs")
+    for library in cves_per_library:
+        cves = library["cves"]
+        # We create the CVEs from older to newer
+        # so that the newer ones are on top of the Github issues list
+        for cve in reversed(cves):
+            if cve.name in opened_cve_issues or cve.name in all_ignored_cves:
+                continue
+
+            library_name = library["name"]
+
+            try:
+                github_api.createIssue(
+                    f"Library {library_name} has vulnerability {cve.name}",
+                    cve.description,
+                    ["cve-%s" % cve.severity.lower(), "libraries"],
```

Comment:
Are these the labels? If so, I'd recommend:
1. We add `security`
2. maybe split `cve` and `severity` in two

These should make it more searchable